challenges-mars/README.md

```diff
@@ -0,0 +1,73 @@
+# Challenge: Who's on Mars?
+
+## Overview
+
+NASA has sent three rovers to Mars: Spirit, Opportunity, and Curiosity. These rovers move around Mars taking pictures (among other things).
+
+Unfortunately, rovers eventually stop sending back pictures. Some of the rovers stopped sending back pictures already.😥
+
+## Difficulty Level
+
+This is a advanced challenge. Level one can be completed by any student who has done week 3 of the CYF React module.
+
+## API
+
+[This page gives details about NASA’s APIs](https://api.nasa.gov/). You should be ok to use DEMO_KEY as your API key for experimenting, but you may need to register for your own API key. Make sure not to check your API key into git.
+
+You should use the `Mars Rover Photos` API to get data.
+
+Make sure you read the website carefully before starting.
+
+You can use this API to get a list of the Rovers:
+
+e.g. https://api.nasa.gov/mars-photos/api/v1/rovers/?api_key=DEMO_KEY
+
+You can then use this api to get information about the rover
+
+e.g. https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity?api_key=DEMO_KEY
+
+You can then use this API to get photos from that rover:
+
+e.g. https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?sol=1000&api_key=DEMO_KEY
+
+_Note: Earth dates and Mars dates (“sols”) are slightly different. For the UI of this website, we always want to use Earth dates._
+
+## Getting Started
+
+**Don't** clone this repo.
+
+Make your own React app using `create-react-app`. See [(this guide)](https://docs.codeyourfuture.io/students/guides/creating-a-react-app) if you have forgotten how.
+
+Copy across the names file (or its contents) from this repo to your app, and then import from that file.
```

Comment:
What is this? I think it's left over from somewhere this section was copied from?